x-pack/plugins/enterprise_search/server/lib/enterprise_search_request_handler.ts

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import fetch from 'node-fetch';
```

Comment:
V small newline nit suggestion, mostly for consistency w/ other files

```suggestion
 */

import fetch from 'node-fetch';
```